spec/lucky/text_response_spec.cr

```diff
@@ -67,6 +159,11 @@ private def print_response(context : HTTP::Server::Context, status : Int32?)
   print_response_with_body(context, "", status)
 end
 
-private def print_response_with_body(context : HTTP::Server::Context, body = "", status = 200, content_type = "text/html")
+private def print_response_with_body(
+  context : HTTP::Server::Context,
```

Comment:
Did the formatter do this? I ask because the one below has the first arg start on the same line as the signature. I'm thinking maybe we make them the same (one way or the other). I was curious if the formatter allowed both, or if it prefers one vs the other way. Does that make sense?

Reply:
I tend to pit the first argument on a new line if everything doe not fit on the same line. But the formatter does respect that and will put all following arguments on the same line.

I like it because that will consistently put all multi-line arguments on the same indentation throughout the file. Otherwise, they are hanging at different places, sometimes with odd numbered indentation levels, depending on the length of the name of the method. 

Here again, if you prefer the first argument at the same line as the method, I'll change that. ☺️

Reply:
BTW, I have been moving a lot of code around there, because that's where I initially added the changes. But then I realised it should be somewhere else. That's why it has been changed.

Reply:
I think this look good 👍

Reply:
I'm ok with doing this. Should we do the same on https://github.com/luckyframework/lucky/pull/1160/files#diff-a949f27bf5c58356d03de361bcf6cba2R182 as well?

Reply:
That would be cleaner indeed, so I added those changes as well.